public/video-ui/src/components/VideoAssets/VideoAssets.js

```diff
@@ -110,14 +110,14 @@ class VideoAssets extends React.Component {
   renderHeader() {
     const buttons = <div className="video-assets__buttons">
       <a className="button" onClick={this.showAssetForm}>
-        <Icon className="icon__edit" icon="add"/>
+        <Icon className="icon__edit icon__spacing" icon="add"/>
```

Comment:
yep, but the upload button is a sibling of the `a` rather than the `Icon`

it is a fairly minor point tbh, but makes the css simpler imo